src/TConsole.cpp

```diff
@@ -563,8 +564,8 @@ Host* TConsole::getHost()
 
 void TConsole::resizeConsole()
 {
-    QSize s = QSize(width(), height());
-    QResizeEvent event(s, s);
+    QSize size = QSize(width(), height());
+    QResizeEvent event(size, size);
```

Comment:
Would you believe that one of those is supposed to be the previous/*old* size - I've never really worked out what faking it means here...

:man_shrugging:

Reply:
Yeah, this bit of code should have a comment explaining the "why"...